deployments/kubehound/graph/Dockerfile

```diff
@@ -59,9 +59,22 @@ ENV gremlinserver.metrics.slf4jReporter.enabled=false
 ENV gremlinserver.metrics.graphiteReporter.enabled=false
 ENV gremlinserver.metrics.csvReporter.enabled=false
 
+# Add safety net settings to prevent OOM and other issues
+ENV janusgraph.query.force-index=true
```

Comment:
I think we probably want to avoid that blocking mode in the open source version. Most users will have a local running version with much smaller cluster than we are using internally, this would add more complexity for some queries that don't necessarly want to use indexes, for something they "can just wait a second".
We can probably enforce it internally if we have performance issues, but I don't think we should block end user doing query by default. (But we should document it if users need that capability)

Reply:
It's not a performance-related flag, but it is more server-resiliency-oriented. We don't block the end user from doing queries; we enforce indices usage to prevent full graph scan, which will consume/kill the server. That's a recommended setting for production usage.